site/docs/uk/plugins/entity-parser.md

```diff
@@ -0,0 +1,238 @@
+---
+prev: false
+next: false
+---
+
+# Парсер сутностей (`entity-parser`)
+
+Перетворює [сутності Telegram](https://core.telegram.org/bots/api#messageentity) у семантичний HTML.
+
+## Коли мені слід використовувати це?
+
+Напевно, ніколи!
+
+Хоча цей плагін може генерувати HTML, зазвичай краще надсилати до Telegram безпосередньо текст і сутності.
+
+Перетворення в HTML необхідне лише в рідкісних випадках, коли вам потрібно використовувати текст у форматі Telegram **поза межами** самого Telegram, наприклад, для відображення повідомлень Telegram на вебсайті.
+
+Перегляньте розділ [_Випадки, коли краще не використовувати цей пакет_](#випадки-коли-краще-не-використовувати-цеи-пакет), щоб визначити, чи є у вас подібна проблема, яку потрібно вирішити.
+
+Якщо ви не впевнені, чи підходить цей плагін для вашого випадку використання, будь ласка, не соромтеся запитувати в нашій [групі в Telegram](https://t.me/grammyjs).
+У більшості випадків люди приходять до висновку, що їм насправді не потрібен цей плагін для вирішення їхніх проблем!
+
+## Встановлення
+
+Запустіть наступну команду у вашому терміналі на основі вашого середовища виконання або пакетного менеджера:
+
+::: code-group
+
+```sh:no-line-numbers [Deno]
+deno add jsr:@qz/telegram-entities-parser
+```
+
+```sh:no-line-numbers [Bun]
+bunx jsr add @qz/telegram-entities-parser
+```
+
+```sh:no-line-numbers [pnpm]
+pnpm dlx jsr add @qz/telegram-entities-parser
+```
+
+```sh:no-line-numbers [Yarn]
+yarn dlx jsr add @qz/telegram-entities-parser
+```
+
+```sh:no-line-numbers [npm]
+npx jsr add @qz/telegram-entities-parser
+```
+
+:::
+
+## Просте використання
+
+Використовувати цей плагін дуже просто.
+Ось короткий приклад:
+
+```ts
+import { EntitiesParser } from "@qz/telegram-entities-parser";
+import type { Message } from "@qz/telegram-entities-parser/types";
+
+// Для кращої продуктивності створіть екземпляр поза функцією.
+const entitiesParser = new EntitiesParser();
+const parse = (message: Message) => entitiesParser.parse({ message });
+
+bot.on(":text", (ctx) => {
+  const html = parse(ctx.msg); // Перетворення тексту в рядок HTML.
+});
+
+bot.on(":photo", (ctx) => {
+  const html = parse(ctx.msg); // Перетворення підпису в рядок HTML.
+});
+```
+
+## Просунуте використання
+
+### Налаштування вихідного тегу HTML
+
+Цей пакет перетворює сутності у семантичний HTML, дотримуючись найкращих практик і стандартів, наскільки це можливо.
```

Comment:
```suggestion
Цей пакет перетворює сутності у семантичний HTML, намагаючись дотримуючись найкращих практик і стандартів.
```

Reply:
I don't see how this version is better than the previous one.

Reply:
> I don't see how this version is better than the previous one.

It is more concise and clear.